pkg/app/ops/mysqlensurer/schema.sql

```diff
@@ -5,9 +5,9 @@
 CREATE TABLE IF NOT EXISTS Project (
   Id BINARY(16) PRIMARY KEY,
   Data JSON NOT NULL,
+  Extra VARCHAR(100) GENERATED ALWAYS AS (data->>"$._extra") STORED UNIQUE,
```

Comment:
I have just realized why do we need UNIQUE for this field?

Reply:
Nice catch 😂 I will remove it 👍

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/1774/commits/ebb982a026739a45b4e6e264dd79f127dfca2687 🙏